db/migrate/20230411144323_add_content_attributes_to_channel.rb

```diff
@@ -0,0 +1,9 @@
+class AddContentAttributesToChannel < ActiveRecord::Migration[6.1]
+  def up
+    add_column :channel_facebook_pages, :content_attributes, :jsonb, default: {}
+  end
+
+  def down
+    remove_column :channel_facebook_pages, :content_attributes, :jsonb
+  end
+end
```

Comment:
not required?

Reply:
Oh yes we could remove this column now! We are not saving information in the channel now.